config/city-lists.json

```diff
@@ -43,9 +73,24 @@
 		"img": "/img/london.gif",
 		"address": "155 Bishopsgate, London EC2M 3YD",
 		"map": "https://maps.app.goo.gl/b2Vb5H2mM41F9nQA6",
-		"sponsors": [
-			"/img/apidays.png"
-		],
+		"sponsors": {
+			"eventSponsors" : [{
+				"image":"/img/apidays.png",
+				"websiteUrl":"https://www.apidays.global/"
+			}],
+			"financialSponsors" : [
+				{
+					"image":"/img/graviteeio.svg" , 
+					"websiteUrl": "https://www.gravitee.io/",
+					"className":"w-[250px] h-[50px]" 
```

Comment:
please share a video of your testcases working, ie failing and passing.

Reply:
Landing.cy.js
![Screenshot from 2024-11-27 23-26-53](https://github.com/user-attachments/assets/e9db8c0b-bcf8-4a6f-ba17-381f46d50810)

Venue.cy.js
![image](https://github.com/user-attachments/assets/fed6cade-70fe-48f9-8380-879898cf6259)

Reply:
I meant the image visiblity test you added, show that it works.

Reply:
Yes as you can see in the image ```should contain logos in Sponsor component``` test that i have written are working